packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -733,9 +745,11 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 				this.logger.sendTelemetryEvent({
 					eventName: "DeltaManager_GetDeltasAborted",
 					fetchReason,
+					// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 					reason: controller.signal.reason,
 				});
 			}
+			// eslint-disable-next-line unicorn/no-null, unicorn/prefer-add-event-listener
```

Comment:
Seems like we could put the listener lambda in a local variable to use `removeEventListener` here... but the check for `onabort === null` above leaves me a bit concerned, so probably best to do this for now.

Reply:
Yeah, agreed. I looked at this very briefly, and determined I didn't have enough context to make a change here.